sklearn/preprocessing/_encoders.py

```diff
@@ -735,6 +739,13 @@ class OrdinalEncoder(_BaseEncoder):
     dtype : number type, default np.float64
         Desired dtype of output.
 
+    encode_missing : boolean, default False
```

Comment:
Talking with @jorisvandenbossche we thought we could create more consistency with proposed changes to OneHotEncoder by having this parameter be named `handle_missing`. It would take values `retain` (default) and `encode`.

Reply:
Okay, great. I'll add that.